app/src/main/java/com/sopt/now/compose/presentation/login/LoginScreen.kt

```diff
@@ -59,20 +63,31 @@ fun LoginScreen() {
     val getResult =
         rememberLauncherForActivityResult(contract = ActivityResultContracts.StartActivityForResult()) { result ->
             result.data?.run {
-                if (result.resultCode == Activity.RESULT_OK) {
-                    user = getParcelableExtra(USER_DATA) ?: User("", "", "", "")
+                if (result.resultCode != Activity.RESULT_OK) context.toast(context.getString(R.string.toast_login_sign_fail))
+            }
+        }
+
+    LaunchedEffect(loginViewModel.postLogin, lifecycleOwner) {
+        loginViewModel.postLogin.observe(lifecycleOwner) {
+            when (it) {
+                is UiState.Success -> {
+                    context.toast(it.data.toString())
+                    Intent(context, HomeActivity::class.java).apply {
+                        flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
+                    }.let { context.startActivity(it) }
```

Comment:
람다 함수를 사용하면 함수를 새로 하나 만드는 건데요, 아래처럼 익명 함수를 하나 만들지 않고 context의 함수로 바로 연결시켜주는 것이 약간의 성능 향상에 도움이 돼요!
```suggestion
                    Intent(context, HomeActivity::class.java).apply {
                        flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
                    }.let(context::startActivity)
```

Reply:
::를 많이 써보지 않아 생각해보지 못했던 부분이네요! 적용하겠습니다!